### ISSUE TITLE
WCAG 2.2 errata  MathML version of the relative luminance definition

### ISSUE THREAD

**Issue by rscano** [w3c/wcag#3490]
https://www.w3.org/TR/WCAG22/relative-luminance.html

There are several issues in the linked file:

- no language declaration
- broken link for SRGB that points to http://github/w3c/wcag/guidelines/#bib-srgb instead of https://w3c.github.io/wcag/guidelines/22/#bib-srgb
- broken link to relative luminance that points to https://www.w3.org/TR/WCAG22/#relativeluminancedef instead of #dfn-relative-luminance

there are also other error of conformance: https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.w3.org%2FTR%2FWCAG22%2Frelative-luminance.html

**Comment by fstrr** [comment]
1. The [language declaration was fixed recently](https://github.com/w3c/wcag/pull/3467) and merged into main. If it's not live yet, it should be soon.
2. The broken link shouldn't be pointing to github.io pages as those are preview pages. The link should go to: https://www.w3.org/TR/WCAG22/#bib-srgb
3. Other errors of conformance: PR 3467 I linked to also removed the xml declaration. The trailing slashes listed are likely needed as part of the current build process for the site.

I've created [PR 3490](https://github.com/w3c/wcag/pull/3507) to fix the two broken links.
